KataVista/utils/Description/Tips.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace BedrockStreamingUtils\CodingDojo\KataVista\Description;
+
+class Tips
+{
+    public const HELPER_SIMPLE = 'Do not try to overthink things, you can always refactor later.';
+    public const HELPER_MULTIPLE = 'You can use multiple helpers inside a single test to check all the new rule properties.';
+    public const HELPER_POSITION = 'There is a helper function to check that a landmark is in a specific position (right/left).';
+    public const HELPER_GROUND = 'There is a helper function to check that a landmark is in a specific plan (foreground/middleground/background).';
+    public const HELPER_NUMBER = 'There is a helper function to check that a landmark is present multiple times.';
+    public const HELPER_STACK = 'There is a helper function to check that a landmark is stacked over another one.';
+    public const HELPER_RELATIVE = 'The new rule can help  position previously discovered elements. This might be the time for some refactoring in the Painter.';
```

Comment:
:art: 
```suggestion
    public const HELPER_RELATIVE = 'The new rule can help position previously discovered elements. This might be the time for some refactoring in the Painter.';
```

Reply:
Fixed :bow: